fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -593,7 +593,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
 
     // this is used to make c3p0 library happy
     @VariableMgr.VarAttr(name = MAX_ALLOWED_PACKET)
-    private int maxAllowedPacket = 1048576;
+    private int maxAllowedPacket = 33554432; // 32MB
```

Comment:
maybe a few comments to explain the usage of this variable, so it won't be taken as NOT-USED from code perspective?

Reply:
https://github.com/mysql/mysql-connector-j/blob/8.0.33/src/main/core-impl/java/com/mysql/cj/NativeSession.java#L428

jdbc source code that uses `max_allowed_packet` session variables.

Reply:
and possibly add ut to use jdbc connect to the PseudoCluster, simulate the `PacketTooBigException` :)

Reply:
documentation will be updated after pr merged

Reply:
comments updated